src/sql/workbench/contrib/connection/browser/connectionStatus.ts

```diff
@@ -104,7 +104,7 @@ export class ConnectionStatusbarItem extends Disposable implements IWorkbenchCon
 	 * Helper function for getting the connection info of the current editor.
 	 */
 	private getCurrentActiveEditorInfo(): ConnectionManagementInfo | undefined {
-		if (this._editorService.activeEditor) {
+		if (this._editorService.activeEditor && this._editorService.activeEditor.resource) {
```

Comment:
You can just do optional chaining instead of writing 2 conditions

this._editorService?.activeEditor?.resource. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining